react/components/ShowAvailable.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+
+import { useContextCssHandles } from './CssHandlesContext'
+
+interface Props {
+  showAvailabilityMessage?: string
+  availableQuantity: number
+}
+
+function ShowAvailable({ showAvailabilityMessage, availableQuantity }: Props) {
+  const { handles } = useContextCssHandles()
+
+  return (
+    <span className={`${handles.showAvailableText} c-muted-2 t-body b mh1`}>
+      {showAvailabilityMessage} {availableQuantity}
+    </span>
```

Comment:
Make this `span` a `Fragment` and separate each part of the message in its own `span` tag with individual handles. You can take a look at the `LowStock` component for inspiration.

I also think you should use the `LowStock` approach of splitting the string on `{quantity}`. That give users more freedom to insert text before and after the quantity and also makes it consistent with the existing API for this component.